CloudM.Exercise.WebApi/PackageDependencies.cs

```diff
@@ -6,5 +6,5 @@ public class PackageDependencies
 
     public string Version { get; set; }
 
-    public Dictionary<string, string> Dependencies { get; set; }
+    public Dictionary<string, dynamic> Dependencies { get; set; }
```

Comment:
I don't think this should be dynamic, because we have the structure of an object that stores these values.